api/metrics/permissions.py

```diff
@@ -8,3 +9,11 @@ def has_permission(self, request, view):
         if user.system_tags.filter(name='preprint_metrics').exists():
             return True
         return False
+
+class IsInstitutionalMetricsUser(permissions.BasePermission):
```

Comment:
There should be tests for this. It appears to always pass, I hacked my way around it, but [the example here](https://github.com/CenterForOpenScience/osf.io/pull/9318/files#diff-3686cc09255e0ed94894729ac8b2c002R13-R23) I'm pretty sure is overriding `has_permission` in a wrong way.